### DISCUSSION TITLE
Stress based damage model

### DISCUSSION THREAD

**Discussion by avtarsinghh1991** [idaholab/moose#23013] [Q&A Modules: General] [answered]
Hello MOOSE Experts,

I want to implement a simple stress based damage model for my simulation.

For example,
if von-Mises stress for any element >= threshold value
damage =1
else
damage=0
end

Can you please locate me a suitable example to do so?
I shall be very thankful.

Best,
Avtar

**Comment by GiudGiud** [comment, marked as answer]
Hello

Are you looking at cracking?
Stress thresholds are used for some of the models there, you could take a look
https://mooseframework.inl.gov/moose/source/materials/ComputeSmearedCrackingStress.html

Guillaume

**Reply by jiangwen84** [reply]
And also look at DamageBase class.
